### PR TITLE
[VEN-2207][VEN-2208]: Src and Remote chain XVS bridge configuration for opBNB

### PR DESCRIPTION
**Pull request by GitGuru7**
Resolves VEN-2207, VEN-2208

### CODE REVIEW

File: vips/vip-216/vip-216-testnet.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+      },
+      {
+        target: XVSBridgeAdmin_Proxy,
+        signature: "setMaxDailyReceiveLimit(uint16,uint256)",
+        params: [DEST_CHAIN_ID, MAX_DAILY_RECEIVE_LIMIT],
+      },
+      {
+        target: XVSBridgeAdmin_Proxy,
+        signature: "setMaxSingleReceiveTransactionLimit(uint16,uint256)",
+        params: [DEST_CHAIN_ID, "10000000000000000000"],
```
[the remark this answers is not in the document]
Comment (GitGuru7) [reply vips/vip-216/vip-216-testnet.ts]: [c071440](https://github.com/VenusProtocol/vips/pull/123/commits/c0714405690a8861fb8e0706035cc9d889eed492)

File: vips/vip-216/vip-216-testnet.ts [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+      },
+      {
+        target: XVSBridgeAdmin_Proxy,
+        signature: "setMaxDailyLimit(uint16,uint256)",
+        params: [DEST_CHAIN_ID, MAX_DAILY_SEND_LIMIT],
+      },
+      {
+        target: XVSBridgeAdmin_Proxy,
+        signature: "setMaxSingleTransactionLimit(uint16,uint256)",
+        params: [DEST_CHAIN_ID, "10000000000000000000"],
```
[the remark this answers is not in the document]
Comment (GitGuru7) [reply vips/vip-216/vip-216-testnet.ts]: [c071440](https://github.com/VenusProtocol/vips/pull/123/commits/c0714405690a8861fb8e0706035cc9d889eed492)

File: multisig/proposals/vip-003/vip-003-opbnbtestnet.ts [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+    { target: XVS_BRIDGE_ADMIN, signature: "acceptOwnership()", params: [] },
+    {
+      target: XVS_BRIDGE_ADMIN,
+      signature: "setTrustedRemoteAddress(uint16,bytes)",
+      params: [10102, "0x963cAbDC5bb51C1479ec94Df44DE2EC1a49439E3"],
+    },
+    {
+      target: XVS,
+      signature: "setMintCap(address,uint256)",
+      params: [XVS_BRIDGE, "100000000000000000000"],
```
[the remark this answers is not in the document]
Comment (GitGuru7) [reply multisig/proposals/vip-003/vip-003-opbnbtestnet.ts]: [c071440](https://github.com/VenusProtocol/vips/pull/123/commits/c0714405690a8861fb8e0706035cc9d889eed492)

**Comment by GitGuru7** [comment]
Src: https://testnet.bscscan.com/tx/0x6264803d58af02c9dd450d6c5698b43d62a8d1e9054c9ba19aadebb517c0fe85

**Comment by GitGuru7** [comment]
bsctestnet -> opBnbTestnet https://testnet.layerzeroscan.com/tx/0x74e469c66febf876f6210b0f15dab550191144689b32bb361d48ff247872bc4d

**Comment by chechu** [comment]
The second testnet VIP (currently `vips/vip-216/vip-216-testnet-2.ts`) and the second testnet multisig TX (currently `multisig/proposals/vip-003/vip-003-opbnbtestnet-2.ts`) to set the new limits related to opbnbtestnet lgtm

**Comment by GitGuru7** [comment]
vip-003-opbnbtestnet-2 : https://testnet.opbnbscan.com/tx/0x8fe097cf788cf8a4721c165541a2e64808e7dca8701706153fe4b71ba0f9e95d

vip-216-testnet-2 : https://testnet.bscscan.com/tx/0x8ea173f95594fcf67931298987b05c2a45c231d7696a78a54e14cf289e12f43c

**Review by chechu — COMMENTED** [review]
opBNB mainnet multisig TX and BNB mainnet VIP lgtm

**Review by kkirka — APPROVED** [review]

**Review by chechu — APPROVED** [review]

**Comment by chechu** [comment]
https://app.venus.io/#/governance/proposal/251?chainId=56

**Comment by chechu** [comment]
Dest TX on opBNB mainnet: https://opbnbscan.com/tx/0xdf8b98f42d39ca4a84d49fb1ebc9f62630436bfbcef79f9cebff254c2338e02f?tab=logs

**Pull request merged**